GoogleForCreators/web-stories-wp
# Improve story appearance on the revisions screen

## Context



The goal is to improve the appearance of stories on the revisions screen.

## Summary



Uses `<amp-story-player>` on the revisions screen to improve the UX for comparing revisions.

## Relevant Technical Choices



## To-do



- [ ] Add more PHP tests

## User-facing changes



<img width="1550" alt="Screenshot 2022-09-12 at 15 30 15" src="https://user-images.githubusercontent.com/841956/189777087-39141626-0e19-47a3-82ab-c9b44cfc59aa.png">

## Testing Instructions






This PR can be tested by following these steps:

1. Enable improved autosaves experiment
2. Create a new story with some content
3. Publish story
4. Make some changes to the story and save it
5. Click on the revisions link in the Document tab
6. Verify that the revisions page shows the embedded before/after stories

## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #12135
